### DISCUSSION TITLE
How to select the "main" photo for a stack?

### DISCUSSION THREAD

**Discussion by rgilton** [immich-app/immich#8498] [Feature Request]
When one has a stack of images, how do you select the "main" image to use to represent the stack?

In the web interface, if I have the stack open, then I can click on the thumbnails of each photo.  When I click on one of the thumbnails, a white dot appears beneath it.  This _looks_ like it is maybe supposed to select the main image, but the selection does not persist when I exit viewing the stack.

**Comment by NicholasFlamy** [comment]
As far as I can tell, the White dot simply shows the currently selected image. I don't see any way to change the main image. I'll make this a feature request.

**Comment by jrasm91** [comment]
There is an update stack API call which supports changing the "primary" asset in a stack, but it doesn't look like there is a way to do that action on the web interface yet. Should be easy to add though.

**Comment by belst** [comment]
I noticed that when selecting multiple images to stack, the one I select first is always selected as the primary

**Reply by Aevaris** [reply]
I noticed this as well, although once created there isnt a gui way to change the main photo without re-creating the stack.

**Reply by develth** [reply]
no always, after doing a de-stack it gets irretaded, so you have to de-stack again and then select. Also selection via tool is not important / does not matter. It just puts it randomly

**Comment by xCJPECKOVERx** [comment]
This was added in #18913 and can be closed
